### ISSUE TITLE
Store db config in user's project

### ISSUE THREAD

**Issue by leanne1** [buildit/storybook-addon-blabbr#49]
Currently the DB config is stored in the npm package. However, each user will need to supply their own config for this so we need a way of keeping this config local to the user.

**Comment by leanne1** [comment]
These are some options:

1. Hard code the config key-value pairs in a pre-start npm script and pass to storybook - this is not feasible because `package.json` must be checked in but db config should not be
2. Pass a value with the config path to storybook to tell it where to find the config. This won't work with ES6 imports which cannot be dynamic. I think Webpack does allow dynamic requires TO LOOK INTO
3. In a given env when we are using an install version of storybook just assume the config with always be in ~/.storybook/config.blabbr.db.js - this is a little brittle but simple
4. Create a bash script that contains the config which npm run storybook calls initially - would need to be OS agnostic and requires the user to have another file....
5. Place the config in a prescribed folder and have webpack default to looking in that folder for non-path imports

**Comment by leanne1** [comment]
None of the above will work. We export dist code to the end user in `node_modules`. By then all modules in blabber package have been bundled and imports resolved. So the DB config must be part of that dist code by the time its ready to install....

**Comment by leanne1** [comment]
**5**.
Works correctly in local dev with the following `webpack.config.js` in `/.storybook`:

```js
const path = require('path');
const webpack = require('webpack');

module.exports = {
	module: {
		preLoaders: [],
		loaders: [
			{ test: /\.(js|jsx)$/, exclude: [ /node_modules/ ], loader: 'babel' },
			{ test: /\.(png|jpg|gif|svg)$/, loader: 'file-loader' },
                        { test: /\.css$/, loader: 'style-loader!css-loader' },
			{ test: /\.(eot|ttf|svg|woff|woff2)(\?v=\d+\.\d+\.\d+)?$/, loader: 'file-loader' }
		]
	},
	resolve: {
		root: [path.resolve(__dirname), path.resolve(__dirname) + '.storybook'],
		extensions: ['', '.js', '.jsx'],
	},
	plugins: [
	],
	devServer: {
		stats: 'minimal'
	},
	devtool: 'source-map'
};
```

And a `config.blabbr.db.js` placed in `/.storybook` and `src/components/api/db.js` importing config with `import config from 'config.blabbr.db';`.

However, when the package is installed in the storybook playground, a `config.blabbr.db.js` placed in `/.storybook`, and the following `webpack.config.js` placed in `/.storybook`, node throws errors:

```js

var genDefaultConfig = require('@kadira/storybook/dist/server/config/defaults/webpack.config.js');

module.exports = function(config, env) {
	var config = genDefaultConfig(config, env);
	config.resolve = {
		root: [path.resolve(__dirname), path.resolve(__dirname) + '.storybook'],
		extensions: ['', '.js', '.jsx'],
	};
	config.node = {
		fs: "empty"
	};
	return config;
};
```

Giving up on this one!

**Comment by panosvoudouris** [comment]
Could we not just do it by calling some config method in .storybook/config.js along with the rest of the config steps around where we do the addDecorator?

**Comment by leanne1** [comment]
I've done this so that the config is passed into the story. But we need the config in the panel. We could emit a message from story -> panel with the config but this feels like a very roundabout way of doing it. I can't yet find an obvious way of passing config directly to a panel...

**Comment by panosvoudouris** [comment]
Resolved in 0.2.1
